manuscript/book.md

```diff
@@ -418,9 +418,9 @@ return getProducts().then(({ products }) =>
 );
 ```
 
-Condition is still here but the overall code structure is simpler.
+A condition is still here but the overall code structure is simpler.
 
-Guard clauses or early return is a great way to avoid nested conditions, also known as the [arrow anti pattern](http://wiki.c2.com/?ArrowAntiPattern) or dangerously deep nesting. Nested conditions are often used for error handing:
+Guard clauses (a.k.a., "early return") is a great way to avoid nested conditions (a.k.a., the [arrow anti pattern](http://wiki.c2.com/?ArrowAntiPattern) or dangerously deep nesting). Nested conditions are often used for error handing:
```

Comment:
```suggestion
Guard clauses, or early return, is a great way to avoid nested conditions, also known as the [arrow anti pattern](http://wiki.c2.com/?ArrowAntiPattern) and dangerously deep nesting. Nested conditions are often used for error handing:
```